### PR TITLE
Feature/mobile width

### PR DESCRIPTION
**Pull request by TheLeoB**
- [x] Spot checked

### CODE REVIEW

**Comment by TheLeoB** [comment]
https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/development/packages/blockchain-info-components/src/Text/TextGroup.spec.js <- a test is failing here and I don't know why.

File: packages/blockchain-wallet-v4-frontend/src/components/Faq/FaqContent/index.js [hunk 1]
```diff
@@ -25,11 +24,11 @@ const FaqContent = [
...
     }, {
-      question: <FormattedMessage id='scenes.faq.group.walletfunctionality.question4' defaultMessage='Why are my bitcoin and bitcoin cash addresses changing?' />,
-      answer: <FormattedMessage id='scenes.faq.group.walletfunctionality.answer4' defaultMessage='Although you can reuse addresses, transacting with the same address makes it easy for people to track your payments history. We alleviate this by using a HD (hierarchical deterministic) framework which provides you endless different addresses to send and receive to for additional security.' />
+      question: <FormattedMessage id='scenes.faq.group.walletfunctionality.question4' defaultMessage={'What is the difference between a wallet ID and an address?'} />,
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/components/Faq/FaqContent/index.js]: Does the translation file pick up the contents of curly braces?

File: packages/blockchain-wallet-v4-frontend/src/scenes/Settings/Security/template.js [hunk 2]
```diff
@@ -8,8 +8,11 @@ import BasicSecurity from './BasicSecurity'
 import AdvancedSecurity from './AdvancedSecurity'
 
 const Wrapper = styled.section`
-  padding: 30px;
+  padding: 20px;
   box-sizing: border-box;
+  @@media (min-width: 480px) {
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Settings/Security/template.js]: Can you fix this, I don't think @@media is valid

File: packages/blockchain-wallet-v4-frontend/src/components/Security/SecurityContainer.js [hunk 3]
```diff
@@ -7,6 +7,10 @@ const SecurityContainer = styled.div`
   border: 1px solid ${props => props.theme['gray-2']};
   border-radius: 4px;
   padding: 20px;
+    @media (min-width: 400px) and (max-width: 991px) {
+      display: block;
+      width: 95%;
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/components/Security/SecurityContainer.js]: 95% makes this overflow the parent slightly, `width: auto;` looks good though

File: packages/blockchain-wallet-v4-frontend/src/components/Faq/template.js [hunk 4]
```diff
@@ -40,14 +40,14 @@ const Faq = (props) => {
 
   return (
     <Fragment>
-      <ModalHeader onClose={() => handleTrayRightToggle()}>
-        <FormattedMessage id='layouts.wallet.trayright.faq' defaultMessage='Frequently Asked Questions'/>
+      <ModalHeader onClose={() => handleTrayRightToggle()} >
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/components/Faq/template.js]: Small thing and doesn't matter but the space is only necessary before the closing tag.

**Comment by plondon** [comment]
The reason the tests are failing is because you updated the TextGroup component and there is a snapshot test for that. Run `yarn test:components:update` then commit the changes.

**Pull request merged**